ext/googletest.cmd

```diff
@@ -15,5 +15,5 @@ cd build
 : # on Windows:
 : # https://github.com/google/googletest/blob/main/googletest/README.md#visual-studio-dynamic-vs-static-runtimes
 cmake -G Ninja -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release -DBUILD_GMOCK=OFF -Dgtest_force_shared_crt=ON ..
-ninja
+cmake --build .
```

Comment:
Note that this is a simple change to force all the cache files to be recreated. The keys for the cache files depend on the contents of the ext/*.cmd files. So if we modify any of the ext/*.cmd, the keys will change, and new cache files will be created.